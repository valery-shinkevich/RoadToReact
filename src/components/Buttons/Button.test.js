import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Button  from './Button'

describe('Button', () => {

  it('отрисовывает без ошибки', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Button>Поиск</Button>, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  test('есть корректный снимок', () => {
    const component = renderer.create(
      <Button>Поиск</Button>
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

})
