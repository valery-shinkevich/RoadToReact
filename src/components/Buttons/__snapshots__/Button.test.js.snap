// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button есть корректный снимок 1`] = `
<button
  className=""
  type="button"
>
  Поиск
</button>
`;
